src/components/People/People.tsx

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import cn from 'classnames';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+
+enum Message {
+  peopleLoadingError = 'Something went wrong',
+  noPeopleMessage = 'There are no people on the server',
+}
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<Message | null>(null);
+  const [dataLoader, setDataLoader] = useState(false);
+
+  const { slug } = useParams();
+  const navigate = useNavigate();
+
+  const REDIRECT_TIME = 3000;
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(res => {
+        setPeople([...res]);
+        setDataLoader(true);
+      })
+      .catch(() => {
+        setError(Message.peopleLoadingError);
+        setTimeout(() => navigate('/'), REDIRECT_TIME);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (dataLoader && people.length === 0) {
+      setError(Message.noPeopleMessage);
+    }
+  }, [dataLoader, people]);
+
+  const canShowTable: boolean = !error && dataLoader;
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {loading && <Loader />}
+
+        {error === Message.peopleLoadingError && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {error === Message.noPeopleMessage && (
+          <p data-cy="noPeopleMessage">There are no people on the server</p>
```

Comment:
Why you not use enum in this case?  Fix it for all places. 
```suggestion
          <p data-cy="noPeopleMessage">{Message.noPeopleMessage}</p>
```